src/dune/config.ml

```diff
@@ -206,6 +221,9 @@ let decode =
   and+ cache_check_probability =
     field "cache-check-probablity" Dune_lang.Decoder.float
       ~default:default.cache_check_probability
+  and+ cache_duplication =
+    field "cache-duplication" Caching.Duplication.decode
```

Comment:
Just realised that we forgot to add `Syntax.since Stanza.syntax (2, 0,0)` to the cache related fields.